drivers/wdm/audio/legacy/wdmaud/mmixer.c

```diff
@@ -769,6 +769,27 @@ WdmAudMidiCapabilities(
         return STATUS_UNSUCCESSFUL;
 }
 
+NTSTATUS
+NTAPI
+WdmAudGetPosition(
+    _In_ PDEVICE_OBJECT DeviceObject,
+    _In_ PIRP Irp,
+    _In_ PWDMAUD_DEVICE_INFO DeviceInfo)
+{
+    MIXER_STATUS Status;
+    DWORD Position;
```

Comment:
Usually in kernel mode we use ULONG etc. instead of win32-specific names like DWORD.

Reply:
OK, changed it to `ULONG`. :slightly_smiling_face: